### ISSUE TITLE
Imports in the examples broken

### ISSUE THREAD

**Issue by matteoguarrera** [neuraloperator/neuraloperator#341]
`from neuralop.data.datasets import load_darcy_flow_small` this should be `from neuralop.datasets import load_darcy_flow_small` all the examples have this issue :cry:

**Comment by dhpitt** [comment]
Hi @matteoguarrera, could you describe this issue in more detail? #332 refactored the `datasets` module to `data.datasets` and `data.transforms`. `load_darcy_flow_small` should now be located in `neuralop.data.datasets.darcy`. Is the code throwing an error?

**Comment by matteoguarrera** [comment]
You are right, I guess the error is in the release. The repo is ok, but if you pip install the package it throws the error

**Comment by dhpitt** [comment]
Right, we’re planning to do a bigger release soon. Thanks for raising this
issue.

**Comment by arcturusannamalai** [comment]
:)

**Comment by dhpitt** [comment]
This is stale post 1.0.2 release
